### PR TITLE
ThoughtSpot Writer UI

### PR DESCRIPTION
**Pull request by MiroCillik**
UI for ThoughtSpot Writer. It's more or less a DB writer, it uses the same UI as other DB writers with a few changes:

- it doesn't have `null` or `default` values for columns
- it uses SSH username and password for loading data, but it is not a SSH tunnel

### CODE REVIEW

**Comment by MiroCillik** [comment]
Mozno som mohol tie subory rovno prepisat z CS do JS?

**Comment by MiroCillik** [comment]
Aha, okej super, a @kacurez mozes prosim skuknut jak som tam pridal tie "disabledFields", ci to moze byt takto?
Rad by som to do konca tyzdna vypublikoval.

**Comment by ujovlado** [comment]
@MiroCillik Tomas mas myslim dovo, kuknem ti to ... dostanem sa k tomu ale asi az dnes vecer alebo zajtra.

**Comment by MiroCillik** [comment]
@ujovlado dobre, v pohode, zajtra staci

File: src/scripts/modules/wr-db/react/pages/table/ColumnsEditor.coffee [hunk 1]
```diff
@@ -91,6 +83,28 @@ module.exports = React.createClass
...
+          React.createElement Hint,
+            title: 'Nullable Column'
+          ,
+            'Empty strings in the source data will be replaced with SQL '
+            code null, 'NULL'
+            '.'
+        if @props.editingColumns
+          @_createCheckbox()
+
+  _renderDefaultHeader: ->
```
Comment (ujovlado) [inline src/scripts/modules/wr-db/react/pages/table/ColumnsEditor.coffee]: @MiroCillik vyzera, ze tato fn sa nikde nepouziva.

File: src/scripts/modules/wr-db/react/pages/table/ColumnRow.coffee [hunk 2]
```diff
@@ -86,6 +87,9 @@ module.exports = React.createClass
         opKey
 
   _createCheckbox: (property) ->
+    console.log(@props.disabledFields)
```
Comment (ujovlado) [inline src/scripts/modules/wr-db/react/pages/table/ColumnRow.coffee]: :)

**Comment by MiroCillik** [comment]
@ujovlado diky vlado, dobry postreh mas :) Opravil som to.
A to `disabledColumnFields` moze byt neposlane, pouziva to zatial len tento ThoughtSpot writer. Ostatne DB writery to nemaju.

**Pull request merged**